loky/backend/utils.py

```diff
@@ -84,3 +90,232 @@ def _recursive_terminate(pid):
+                limits[clib] = _get()
+            except NotImplementedError:
+                limits[clib] = None
+        return limits
+
+    def openblas_set_num_threads(self, num_threads):
+        if self.openblas is None:
+            raise NotImplementedError("Could not find OpenBLAS library.")
+        self.openblas.openblas_set_num_threads(num_threads)
+
+    def openblas_get_max_threads(self):
+        if self.openblas is None:
+            raise NotImplementedError("Could not find OpenBLAS library.")
+        return self.openblas.openblas_get_num_threads()
+
+    def openmp_gnu_set_num_threads(self, num_threads):
+
+        if self.openmp_gnu is None:
+            raise NotImplementedError("Could not find OpenMP library")
+        self.openmp_gnu.omp_set_num_threads(num_threads)
+
+    def openmp_gnu_get_max_threads(self):
+        if self.openmp_gnu is None:
+            raise NotImplementedError("Could not find OpenMP library")
+        return self.openmp_gnu.omp_get_max_threads()
+
+    def openmp_intel_set_num_threads(self, num_threads):
+        if self.openmp_intel is None:
+            raise NotImplementedError("Could not find OpenMP library")
+        self.openmp_intel.omp_set_num_threads(num_threads)
+
+    def openmp_intel_get_max_threads(self):
+        if self.openmp_intel is None:
+            raise NotImplementedError("Could not find OpenMP library")
+        return self.openmp_intel.omp_get_max_threads()
+
+    def mkl_set_num_threads(self, num_threads):
+        if self.mkl is None:
+            raise NotImplementedError("Could not find MKL libray")
+        self.mkl.MKL_Set_Num_Threads(num_threads)
+
+    def mkl_get_max_threads(self):
+        if self.mkl is None:
+            raise NotImplementedError("Could not find MKL libray")
+        return self.mkl.MKL_Get_Max_Threads()
+
+    def _load_lib(self, module_name):
+        lib_name = find_library(module_name)
+        if lib_name is not None:
+            return ctypes.CDLL(lib_name, use_errno=True)
+        if sys.platform == "darwin":
+            return self._find_with_libc_dyld(module_name)
+        return self._find_with_libc_dl_iterate_phdr(module_name)
+
+    def _find_with_libc_dl_iterate_phdr(self, module_name):
+
+        global _thread_locals
+        _thread_locals._module_path = None
+
+        libc = self._get_libc()
+        if not hasattr(libc, "dl_iterate_phdr"):
+            return
+
+        c_func_signature = ctypes.CFUNCTYPE(
+            ctypes.c_int, ctypes.c_void_p, ctypes.c_size_t, ctypes.c_char_p)
+        c_match_module_callback = c_func_signature(match_module_callback)
+
+        module_name = "lib{}".format(module_name)
+        data = ctypes.c_char_p(module_name.encode('utf-8'))
+        res = libc.dl_iterate_phdr(c_match_module_callback, data)
+        if res == 1:
+            return ctypes.CDLL(_thread_locals._module_path)
+
+    def _find_with_libc_dyld(self, module_name):
+
```

Comment:
please put a docstring to explain that this is expected to work on macOS only and remove the extra blank line.